.github/workflows/smoketestBackend.yml

```diff
@@ -0,0 +1,49 @@
+name: Smoke Test Back-end API
+
+# Run this on pushes to any branch that change a backend file or the workflow definition
+on:
+  push:
+    branches: "**"
+    paths:
+      - "backend/**"
+      - .github/workflows/smoketestBackend.yml
+
+defaults:
+  run:
+    working-directory: backend
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK 1.8
```

Comment:
Is there a reason to stay on Java 8, given that it's EOL?

Reply:
Just making sure people's development environments don't break, since a lot of us have weirdly constrained dev machines. I think I have now checked with everybody, though, so I will go update it throughout the application (just the pipelines and build.gradle, I think?)